hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CombinedBlockCache.java

```diff
@@ -109,8 +109,6 @@ public Cacheable getBlock(BlockCacheKey cacheKey, boolean caching, boolean repea
         }
       } else {
         if (existInL1) {
-          LOG.warn("Cache key {} had block type {}, but was found in L1 cache.", cacheKey,
```

Comment:
Why removing this warn log?

Reply:
This is not related to this PR itself, but it would address HBASE-28364. Since it's a one liner, I decided to wrap it here.